modules/admin_manual/pages/release_notes.adoc

```diff
@@ -19,6 +20,162 @@
+  For new installations, there will be two apps directories so that the bundled apps are distinguishable from the apps that were installed or updated by the administrator. 
+  Existing installations will not change but, generally, xref:installation/apps_management_installation.adoc#using-custom-app-directories[this separation is recommended] in all scenarios, as it makes upgrading easier and less error-prone.
+* *Update the `occ files:scan` `--group` and `--groups` options.* 
+  The `occ files:scan` command is used to scan resources on the storage and make them available in ownCloud. 
+  While previously it could only be used for all or single users and groups of users, you can now also execute it for groups where the group name contains a comma.
+* *Allow administrators to enable/disable medial search for users and groups.* 
+  Medial search is used to get search results when typing keys within a search term in autocomplete fields (e.g. when typing "_ter_" you'll find "Peter"). 
+  Depending on the configuration of available search terms (e.g., attributes from LDAP), search results can deliver better results without medial search. 
+  For these reasons medial search can now be enabled/disabled for user (`'accounts.enable_medial_search'`) and group (`'groups.enable_medial_search'`) search. 
+  See https://github.com/owncloud/core/blob/stable10/config/config.sample.php#L285[config.sample.php] for more information.
+* Added a new occ command, `background:queue:execute`, for running cron jobs manually.
+* Added two new `occ background:queue` commands: `status` and `delete`.
+** `status` lists the current background job queue status
+** `delete` removes a single background job, identified by its id.
+* The version of pear/archive_tar from 1.4.6 to 1.4.7 
+  
+=== Solved Known Issues
+
+* Fixed public link share default expiration behavior https://github.com/owncloud/core/issues/34971[#34971].
+  Previously, when a default expiration date for public links had been set by an administrator (without enforcement option), the default value has been applied upon link creation even when a user removed it.
+  The only way to create a link without expiration date was to subsequently edit it and remove the expiration date. 
+  This has been fixed to work as expected.
+* Better support for international email addresses after Swiftmailer update https://github.com/owncloud/core/issues/34759[#34759]
+* Improved speed of apps list settings page by caching integrity check results https://github.com/owncloud/core/issues/34584[#34584]
+* Improved upgrade speed when migrating avatars from oC < 10 https://github.com/owncloud/core/issues/34592[#34592]
+* Improved performance and memory usage of account sync service https://github.com/owncloud/core/issues/34546[#34546]
+* Store quota overrides in preferences table https://github.com/owncloud/core/issues/34467[#34467] => to clarify with Tom Needham
```

Comment:
"to clarify" comment still needs addressing/removing @tomneedham @pmaier1

Reply:
Please move to "Solved known issues" and add:

"In former versions functionality has been introduced to preserve quota values imported via LDAP attributes against manual changes by the administrator in ownCloud. With this bug fix the functionality works again."

Reply:
currently I'm writing tests for that, its still possible to change the quota locally, but the change will be reverted on user sync. e.g. synced by `occ` or user login

Reply:
"In former versions manual quota changes by the administrator were preserved against those values imported via user backends, such as LDAP. This fix restores such functionality.”

Reply:
Thanks for the feedback.

Reply:
This has now been updated.

Reply:
After some discussion it turned out to be wrong as it's written now. This should be correct:

"In former versions functionality has been introduced to preserve quota values imported via LDAP attributes against manual changes by the administrator in ownCloud user management or via the provisioning API. This functionality works again properly: If you sync accounts from LDAP and have a quota attribute specified in LDAP, each `user:sync` run will set the quota values to the ones from LDAP no matter if they were changed manually."